Kickstarter-iOS/Views/Controllers/FeatureFlagToolsViewController.swift

```diff
@@ -0,0 +1,109 @@
+import Foundation
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class FeatureFlagToolsViewController: UITableViewController {
+  private let viewModel = FeatureFlagToolsViewModel()
+
+  // MARK: - Properties
+
+  private var features = [(Feature, Bool)]()
+  private let reuseId = "FeatureFlagTools.TableViewCell"
+
+  static func insantiate() -> FeatureFlagToolsViewController {
+    return FeatureFlagToolsViewController.init(nibName: nil, bundle: nil)
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    _ = self.tableView
+      |> \.tableFooterView .~ UIView(frame: .zero)
+
+    self.tableView.register(UITableViewCell.self, forCellReuseIdentifier: self.reuseId)
+
+    self.viewModel.inputs.viewDidLoad()
+  }
+
+  override func bindViewModel() {
+    super.bindViewModel()
+
+    self.viewModel.outputs.reloadWithData
+      .observeForUI()
+      .observeValues { [weak self] features in
+        self?.features = features
+
+        self?.tableView.reloadData()
+      }
+
+    self.viewModel.outputs.updateConfigWithFeatures
+      .observeForUI()
+      .observeValues { [weak self] features in
+        self?.updateConfig(with: features)
+      }
+  }
+
+  @objc func switchTogged(_ switchControl: UISwitch) {
+    self.viewModel.inputs.setFeatureAtIndexEnabled(index: switchControl.tag, isEnabled: switchControl.isOn)
+  }
+
+  // MARK: - Private Helpers
+
+  private func updateConfig(with features: Features) {
+    guard let config = AppEnvironment.current.config else { return }
+
+    let updatedConfig = config |> \.features .~ features
+
+    AppEnvironment.updateConfig(updatedConfig)
+
+    self.viewModel.inputs.didUpdateConfig()
+  }
+}
+
+// MARK: - UITableViewDataSource
+
+extension FeatureFlagToolsViewController {
+  override func numberOfSections(in _: UITableView) -> Int {
+    return 1
+  }
+
+  override func tableView(_: UITableView, numberOfRowsInSection _: Int) -> Int {
+    return self.features.count
+  }
+
+  override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+    let cell = tableView.dequeueReusableCell(withIdentifier: self.reuseId, for: indexPath)
+    let feature = self.features[indexPath.row]
+
+    let switchControl = UISwitch(frame: .zero)
```

Comment:
Was the reason for not creating a custom cell with a switch control simplicity here? Looks like we're re-creating the cell every time.

Reply:
Yeah all of this was just to keep it as simple as possible. We're not really optimizing for performance here.